### PR TITLE
Data Import

### PR DESCRIPTION
**Pull request by ash6851**
@karbassi check it out, all the code is import stuff.

### CODE REVIEW

File: coderdojochi/admin.py [hunk 1]
```diff
@@ -109,7 +114,41 @@ def get_last_name(self, obj):
...
-class GuardianAdmin(admin.ModelAdmin):
+class GuardianImportResource(ModelResource):
+    first_name = Field(attribute='first_name', column_name='first_name')
+    last_name = Field(attribute='last_name', column_name='last_name')
+    email = Field(attribute='email', column_name='email')
+    phone = Field(attribute='phone', column_name='phone')
+    zip = Field(attribute='zip', column_name='zip')
+
+    def get_or_init_instance(self, instance_loader, row):
+        return Guardian(user=User()), True
```
Comment (karbassi) [inline coderdojochi/admin.py]: @ash6851 Should this always create a new guardian? What if the guardian exists (e.g. accidentally importing someone who already exists)?

**Comment by ash6851** [comment]
user registration needs to check for previously created user

**Comment by karbassi** [comment]
Related to #378

**Comment by karbassi** [comment]
@ash6851 Can you give me access to your branch so I can commit fixes?

screen shot 2016-10-06 at 11 53 38 pm

**Pull request merged**